WordPress/gutenberg
# ColorPalette: Prevent overflow of custom color button background

## What?

This PR fixes two issues with the custom color buttons in the color palette buttons:

- The checkerboard pattern overflows at the corners
- The checkerboard pattern overlaps the button border

## Why? / How?

As for the corner overflow, it became more noticeable after the radius value was changed from 2px to 4px in #64368.

As for the overlap with the border, I think it looks more natural to shift it by 1px to avoid overlapping.

## Testing Instructions

Visit http://localhost:50240/?path=/docs/components-colorpalette--docs and check the corners of the component.

## Screenshots or screencast 

### Before

![before_1](https://github.com/user-attachments/assets/9ce28d5e-f36a-423b-a278-8bd64256aa0d)
![before_2](https://github.com/user-attachments/assets/526514f9-7d1e-4039-9a5f-d906a8749966)


### After

![after_1](https://github.com/user-attachments/assets/4c53c0db-5c10-459b-8989-317a9eeca1ea)
![after_2](https://github.com/user-attachments/assets/f43d11fd-f0f0-4182-9fbc-de9c4fd66442)
